nx/lib/nx/lin_alg.ex

```diff
@@ -1151,6 +1174,9 @@ defmodule Nx.LinAlg do
     Nx.Defn.Kernel.assert_shape_pattern(tensor, {n, n})
 
     transform(tensor, fn tensor ->
+      type = Nx.type(tensor)
+      Nx.Shared.raise_complex_not_supported(type, "LinAlg.determinant", 1)
```

Comment:
They are indeed not supported? Or not for now?

Reply:
Not for now! The definitions for complex numbers are different (transposes become transpose+conjugate, for instance + correction factors here and there)

Reply:
Should I add "for now" in the message and function name?

Reply:
or "not supported yet"

Reply:
Yeah, I would raise a different message saying "not yet".